### PR TITLE
Set correct return type for insert method

### PR DESCRIPTION
**Pull request by jaapio**
The official php-documentation was even incorrect at this point as
shown in https://3v4l.org/X3h8Q
Insert returns true when successfull. It is unclear if it can ever
return false.

### CODE REVIEW

**Comment by orklah** [comment]
Shouldn't we change the type to true then?

@b1rdex Does that line means it can return void also? https://github.com/php/php-src/blob/7674a542aa870c4cba462cb2a1b68b08f9b0406e/ext/spl/spl_heap.c#L667

**Comment by jaapio** [comment]
It throws an exception before the return. So I think the return will never be hit. It is an internal corrupted state so I guess it will never happen.

**Comment by orklah** [comment]
Arg! I wanted to link this line:
https://github.com/php/php-src/blob/7674a542aa870c4cba462cb2a1b68b08f9b0406e/ext/spl/spl_heap.c#L660

**Comment by jaapio** [comment]
Same applies there to. The return statement will never be hit because of the check in the if statement. Which triggers an error in php that an argument is missing. I don't think the return is executed.

File: SPL/SPL_c1.php [hunk 1]
```diff
@@ -1553,7 +1553,7 @@ public function compare ($priority1, $priority2) {}
          * @param mixed $priority <p>
          * The associated priority.
          * </p>
-         * @return void 
+         * @return bool
```
Comment (b1rdex) [inline SPL/SPL_c1.php]:
```suggestion
         * @return true
```
@andrey-sokolov I suppose you have write access to accept sugesstion too

**Comment by jaapio** [comment]
Thanks for the improvement

**Pull request merged**